client/actions/blogActions.js

```diff
@@ -26,9 +24,7 @@ export function getBlog(body) {
             .then((response) => {
                 dispatch(getAction(response.data))
             })
-            .catch((err) => {
-                console.log("error ====>>>", err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}